### PR TITLE
docs: Add note about linuxmint issue on ubuntu dependencies

### PR DESCRIPTION
**Pull request by fabiobarkoski**
Added a note about the Linux Mint problem discussed in the #1601.

### CODE REVIEW

File: doc/build_instructions/ubuntu.md [hunk 1]
```diff
@@ -15,3 +15,6 @@ from pip:
 ```
 pip3 install cython --break-system-packages
 ```
+
+# Linux Mint Issue
+Linux Mint has a [problem with toml11](https://github.com/SFTtech/openage/issues/1601), since CMake can't find it. To solve this, download the [toml11.zip](https://github.com/SFTtech/openage/files/13401192/toml11.zip), after, run the openage and wait it throw the toml11 error, and then put the files in the needed path. (if the toml11 directory doesn't exist, create it)
```
Comment (heinezen) [inline doc/build_instructions/ubuntu.md:20]: Can you clarify what the path is? It should be the same for all Linux Mint systems I think.

Comment (fabiobarkoski) [reply doc/build_instructions/ubuntu.md:20]: Sure, but at the first time I ran, the `./configure` required the toml11 dir in the `/lib/x86_64-linux-gnu/cmake` and at the second time in the `/usr/lib/x86_64-linux-gnu/cmake`. Put both paths?

Comment (heinezen) [reply doc/build_instructions/ubuntu.md:20]: `/lib` should symlink to `/usr/lib`, so both should be the same. You only need to put the `/usr/lib/...` path in the text then.

Comment (fabiobarkoski) [reply doc/build_instructions/ubuntu.md]: Done

**Review by heinezen — APPROVED** [review]

**Pull request merged**